solr/modules/ltr/src/java/org/apache/solr/ltr/store/rest/ManagedFeatureStore.java

```diff
@@ -161,9 +168,8 @@ public void doGet(BaseSolrResource endpoint, String childId) {
       response.add(FEATURE_STORE_JSON_FIELD, stores.keySet());
     } else {
       final FeatureStore store = getFeatureStore(childId);
-      if (store == null) {
-        throw new SolrException(
-            SolrException.ErrorCode.BAD_REQUEST, "missing feature store [" + childId + "]");
+      if (store.getFeatures().isEmpty()) {
```

Comment:
Is "do get" ever called by the logging model to manage features?
Is the check in the logger necessary or we would arrive at this check anyway?

Reply:
it seems to me that the "doGet" method is called only when you want to retrieve the list of feature stores or a single feature store. So this check is necessary if, for example, you request a nonexistent feature store: 
e.g. http://localhost:8983/solr/<collection>/schema/feature-store/<wrongFeatureStoreName>